buzsaki_lab_to_nwb/huszar_hippocampus_dynamics/converter.py

```diff
@@ -29,6 +49,24 @@ def __init__(self, source_data: dict, verbose: bool = True):
         self.session_folder_path = Path(self.data_interface_objects["Behavior8Maze"].source_data["folder_path"])
         self.session_id = self.session_folder_path.stem
 
+        # Add electrode locations (modeled after yutavcnwbconverter)
+        electrode_chan_map_file_path = self.session_folder_path / "chanMap.mat"
+        chan_map = loadmat_scipy(electrode_chan_map_file_path)
+        xcoords = [x[0] for x in chan_map["xcoords"]]
+        ycoords = [y[0] for y in chan_map["ycoords"]]
+        kcoords = [y[0] for y in chan_map["kcoords"]]
+
+        for channel_id in chan_map["chanMap0ind"]:
```

Comment:
But the channel information also seems to be added here @CodyCBakerPhD @garrettmflynn

Reply:
So to move forward with this, we'd want to remove the channel information here and use the latest `neuroconv` with https://github.com/catalystneuro/neuroconv/pull/491 merged in?

Reply:
What I think is that the highlighted code should work without adding anything else. Work here means adding the channel metadata. That is, this code competing with https://github.com/catalystneuro/neuroconv/pull/494/files and #65 and could be used instead.

#491 is about something else. It eliminate the need of this code:
https://github.com/catalystneuro/buzsaki-lab-to-nwb/tree/master/buzsaki_lab_to_nwb/huszar_hippocampus_dynamics

But this does not conern this PR.

Reply:
Got it. Thank you for the clarification